core/src/main/java/io/questdb/cutlass/pgwire/AbstractTypeContainer.java

```diff
@@ -50,6 +50,14 @@ public void close() {
         }
     }
 
+    public void copyOutTypeDescriptionTypesTo(IntList outTypeDescriptionTypes) {
+        for (int i = 0, n = types.size(); i < n; i++) {
```

Comment:
we probably should add this method to TypesAndInsert directly. The base class is used also for TypesAndUpdates, that is legacy only now